crux-llvm/src/Crux/LLVM/Compile.hs

```diff
@@ -163,6 +168,38 @@ genBitCodeToFile finalBCFileName files cruxOpts llvmOpts copySrc = do
   return finalBCFile
 
 
+-- | A C source file being compiled and evaluated by crux can contain
+-- zero or more lines matching the following:
+--
+-- >   /* CRUCIBLE clang_flags: {FLAGS} */
+-- >   // CRUCIBLE clang_flags: {FLAGS}
```

Comment:
I've seen confusion from people I've shown Crux to about the difference between Crucible and Crux. I suspect this might be slightly clearer if it started with CRUX instead of CRUCIBLE.

There's part of me that wants `clang_flags` to be `CLANG_FLAGS` to be more similar to Make variables.

Reply:
I wavered initially and went with CRUCIBLE because that's the same namespace as the `#include <crucible.h>` extensions, but I actually updated it to support both "CRUX" and "CRUCIBLE".

I also made the "clang_flags" portion be case-insensitive, so if you want CLANG_FLAGS you can have them. :-)